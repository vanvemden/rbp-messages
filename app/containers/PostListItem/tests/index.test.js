/**
 * Test the repo list item
 */

import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import { PostListItem } from '../index';

const renderComponent = (props = {}) =>
  render(
    <IntlProvider locale="en">
      <PostListItem {...props} />
    </IntlProvider>,
  );

describe('<PostListItem />', () => {
  const item = { id: 1, text: 'Hello World!' };

  it('should render a ListItem', () => {
    const { container } = renderComponent({ item });
    expect(container.firstChild).toMatchSnapshot();
  });
});
