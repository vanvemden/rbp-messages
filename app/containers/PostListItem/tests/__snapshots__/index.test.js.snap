// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PostListItem /> should render a ListItem 1`] = `
<li
  class="Wrapper-euo0oy-0 jLvxtc"
>
  <div
    class="Item-sc-3y9mie-0 fKXSkT"
  >
    Hello World!
  </div>
</li>
`;
